metrics/performance/first-meaningful-paint.js

```diff
@@ -18,49 +18,61 @@
 
 const DevtoolsTimelineModel = require('devtools-timeline-model');
 
+const FAILURE_ERROR = new Error('Navigation and first paint timings not found');
```

Comment:
it would be better to make this just `FAILURE_MESSAGE` and have it just be the string, creating the `new Error()` in place